fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/ReprocessLoanTransactionsServiceImpl.java

```diff
@@ -110,6 +114,15 @@ public void processLatestTransaction(final LoanTransaction loanTransaction, fina
         handleChangedDetail(changedTransactionDetail);
     }
 
+    @Override
+    public void updateModelWithoutRecalcOrReprocessAllTransactions(Loan loan) {
+        Optional<ProgressiveLoanInterestScheduleModel> andUpdateModel = interestScheduleModelRepositoryWrapper.getAndUpdateModel(loan,
```

Comment:
I dont get this method... 
We in fact recalculate things (at least the interest part for some days)
We in fact reprocess transactions if model does not exists or behind...